dom.bs

```diff
@@ -6809,38 +6809,39 @@ are:
  <a for=/>shadow root</a>'s <a for=ShadowRoot>mode</a> is "<code>open</code>", and null otherwise.
 </dl>
 
+<p>A <dfn>valid shadow host name</dfn> is:
```

Comment:
Since you want to use it this is probably needed, right?
```suggestion
<p>A <dfn export>valid shadow host name</dfn> is:
```
I'll merge this Monday unless there are concerns.